recipes/sqlalchemy_schemadisplay/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "sqlalchemy_schemadisplay" %}
+{% set version = "1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0a9f26d77be9d92c9564d87cc17668fe141a816036c5f5d7c8cb053b253957e0
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - pango >=1.40.14  # [not win]
```

Comment:
are you sure about this dependency?
Their ``setup.py`` does not show it
https://github.com/fschulze/sqlalchemy_schemadisplay/blob/2d9132ee62d34f38c41131cbef98371d15451626/setup.py#L20
and I didn't see it being imported

Reply:
`The following NEW packages will be INSTALLED:

    ca-certificates:          2019.11.28-hecc5488_0 mirror-conda-forge
    cairo:                    1.14.6-0              mirror-conda-forge
    certifi:                  2019.11.28-py27_0     mirror-conda-forge
    expat:                    2.1.1-1               esss              
    fontconfig:               2.11.1+1-hef9d510_1   esss              
    freetype:                 2.6.3-1               mirror-conda-forge
    gettext:                  0.19.4-0              esss              
    glib:                     2.51.4-0              mirror-conda-forge
    graphviz:                 2.38.0-3              mirror-conda-forge
    harfbuzz:                 1.3.4-0               mirror-conda-forge
    icu:                      56.1-4                mirror-conda-forge
    jpeg:                     9c-h14c3975_1001      mirror-conda-forge
    libffi:                   3.2.1-h4deb6c0_3      mirror-main       
    libgcc:                   5.4.0+4-0             esss              
    libgcc-ng:                9.1.0-hdf63c60_0      mirror-main       
    libiconv:                 1.15-h516909a_1005    mirror-conda-forge
    libpng:                   1.6.27-0              esss              
    libstdcxx-ng:             9.1.0-hdf63c60_0      mirror-main       
    libtiff:                  4.0.2-0               esss              
    libtool:                  2.4.6-h14c3975_1002   mirror-conda-forge
    libxml2:                  2.9.3-8               mirror-conda-forge
    ncurses:                  6.1-hf484d3e_1002     mirror-conda-forge
    openssl:                  1.1.1d-h516909a_0     mirror-conda-forge
    pango:                    1.40.1-0              mirror-conda-forge
    pcre:                     8.36-0                esss              
    pip:                      20.0.2-py27_0         mirror-conda-forge
    pixman:                   0.38.0-h516909a_1003  mirror-conda-forge
    pydot:                    1.4.1-py27_1001       mirror-conda-forge
    pyparsing:                2.4.6-py_0            mirror-conda-forge
    python:                   2.7.15-h5a48372_1009  mirror-conda-forge
    readline:                 8.0-hf8c457e_0        mirror-conda-forge
    setuptools:               44.0.0-py27_0         mirror-conda-forge
    sqlalchemy:               1.0.15-py27_0         esss              
    sqlalchemy_schemadisplay: 1.3-py_0              local             
    sqlite:                   3.30.1-hcee41ef_0     mirror-conda-forge
    tk:                       8.6.10-hed695b0_0     mirror-conda-forge
    wheel:                    0.34.1-py27_0         mirror-conda-forge
    xz:                       5.0.5-0               esss              
    zlib:                     1.2.11-h516909a_1006  mirror-conda-forge

Preparing transaction: ...working... done
Verifying transaction: ...working... failed
Build prefix failed with prefix length 255
WARNING:conda_build.environ:Build prefix failed with prefix length 255
Error was: 
WARNING:conda_build.environ:Error was: 
Placeholder of length '80' too short in package /usr/local/conda-bld/sqlalchemy_schemadisplay_1580406243155/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/bin/pango-view.
The package must be rebuilt with conda-build > 2.0.
`

Reply:
With pango 1.40.1-0 build fails.

Reply:
No, I mean, I didn't see ``pango`` being imported in this project and thei ``setup.py`` does not specify it
and please update your ``conda-build`` :p

Reply:
It does not seem to be a dependency of ``sqlalchemy_schemadisplay``

Reply:
It seems like pango is an indirect dependency and the older version was built with an old conda build which must be rebuilt with conda-build > 2.0. Said that, the solution found was to pin the dependency to a newer version. I am almost sure that removing it, it will fail.

Reply:
My version of conda-build 3.17.8

Reply:
indirect dependency? what do you mean?
So... it is not a dependency of this package then, isn't it?

By the way, I was able to build this package without ``pango`` on ``linux``

Reply:
Try to build it without the ``esss`` channel
One of your packages might restrain some dependencies and you are seeing those errors

Reply:
For example
You are getting ``libgcc 5.4``... ``conda-forge`` is using ``9.2`` to build the packages

Reply:
Please try to delete ``pango`` from the dependency

Reply:
> Try to build it without the `esss` channel
> One of your packages might restrain some dependencies and you are seeing those errors

I will try that, thanks.

Reply:
> Please try to delete `pango` from the run requirements

It worked when removing the local channel. Thanks for that.

Reply:
BTW, do you know what can cause the following error on Circle CI?
`/home/conda/staged-recipes/.circleci/build_steps.sh: line 48: CONFIG: unbound variable